docs/home/whats-new-2-15.md

```diff
@@ -1,94 +1,103 @@
 
-<!-- RUN-7495/RUN11388 Support PSA / SCCs V2 \(security mechanism for pods on K8S/OCP\)-->
+--8<-- "home/whats-new-2-14.md:6:8"
 
-<!-- RUN-9386 Metrics API: Expose predefined metrics-->
-§
-<!-- RUN-9924/RUN-9925  Granular GPU compute time-slicing / Strict GPU compute time-slicing -->
+--8<-- "home/whats-new-2-14.md:15:16"
 
-<!-- RUN-9949 Container RUNtime Interface - No TW story as per ticket -->
+--8<-- "home/whats-new-2-14.md:18:20"
 
-<!-- RUN-9950 Support JFrog Artifactory private repository - No TW story as per ticket -->
+--8<-- "home/whats-new-2-14.md:26:27"
 
-<!-- RUN-10271/RUN-10321 Mark environment for workload type-->
+--8<-- "home/whats-new-2-14.md:29:31"
 
-<!-- RUN-10404/RUN-11747 Submit distributed training-->
+--8<-- "home/whats-new-2-14.md:33:35"
 
-<!-- RUN-10411/RUN-11390 Support self-signed certificates-->
+--8<-- "home/whats-new-2-14.md:45:47"
 
-<!-- RUN-10446/RUN-10447 RUN:ai Hyper-Scale - Scheduler - Phase 1: Adobe scenario-->
+## Improvements
 
-<!-- RUN-10451/RUN-10452 Support new Kubernetes and OpenShift releases - Q3/2023-->
+<!-- ADDLINK RUN-9943/RUN-12176 Nodes - reflect the correct status of the node - add to nodes page the table from the TW ticket -->
+* Improved the readability of the node table to include a more detailed status and its description. For more information, see [Node and Node Pool Status](##node-and-node-pool-status)
 
-<!-- RUN-10486 Project selection is persistent in the  \(V2\)-->
+<!-- RUN-11421/RUN-11508 Consumption report - Cost and bugs-->
+* Improved the Consumption report interface by moving the Cost settings to the *General* settings menu.
 
-<!-- RUN-10493 Stabilize scheduler post Elastic Workloads changes-->
+<!-- RUN-7085/RUN-9480 Installation - Cluster wizard Improvements -->
+* Improved the *Cluster Wizard* form for adding new clusters to your system. We have eliminated the need for a values file and converted it to a helm command. Cluster configurations are preserved during upgrade and are done using the `runaiconfig` file and create a separation between installation-related flags and cluster customization flags. For more information, see [Customize cluster installation.](##customize-cluster-install.md)
 
-<!-- RUN-10602/RUN-10603 GPU Memory Request & Limit-->
+<!-- ADDLINK RUN-9924/RUN-9925  Granular GPU compute time-slicing / Strict GPU compute time-slicing -->
+* Added the ability to configure strict GPU compute time slicing. This gives workloads the exact GPU compute portion based on the requested GPU fraction (GPU Memory Fraction). This creates complete transparency and predictability of the amount of resources (Compute, Memory, etc.) a workload will get from a GPU. For more information, see [GPU Time Slicing](##gpu-time-slicing).
```

Comment:
I think it is worth mentioning in the RN as well that this feature is experimental